### ISSUE TITLE
[v1.2-Elements-Action.ToggleVisibility.json]: 'Here are some images:' text is not accessible by voice over in 'Action.ToggleVisibility.json' card.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7359]
https://microsoft.visualstudio.com/OS/_workitems/edit/37852937
